quota-management/README.md

```diff
@@ -0,0 +1,214 @@
+# Sample Quota Management Strategy for OpenShift
+
+The following is an example of a strategy that might be employed by an Operations team for managing project quotas. This approach uses the concept of _tier profiles_ so as to avoid having to manage many custom quotas for each project/tenant. In this example we have defined the following tier profiles.
+
+| Profile | Project CPU             | Project Memory          | Per Pod CPU                       | Per Pod Memory                         |
+|---------|-------------------------|-------------------------|-----------------------------------|----------------------------------------|
+| Small (default)  | 1 core (burstable to 2) | 6 Gi (burstable to 8)   | 10m - 100m; default request: 50m  | 128 Mi - 1 Gi; default request: 256 Mi |
+| Medium  | 2 core (burstable to 4) | 12 Gi (burstable to 16) | 10m - 200m; default request: 50m  | 128 Mi - 2 Gi; default request: 256 Mi |
+| Large   | 4 core (burstable to 8) | 24 Gi (burstable to 32) | 10m - 1000m; default request: 50m | 128 Mi - 4 Gi; default request: 256 Mi |
+
+The specs above are not important, nor are the number of profiles we've defined. What is important is how we define & apply the quotas to projects. We describe the process below.
+
+## Concepts and Workflow
+
+The following sections describe how we piece together this approach.
+
+### Quota Files
+
+First off, each quota is definied in its own YAML file (`./quota-small.yml`, `./quota-medium/yml`, `./quota-large.yml`) in a git repo. Each file contains the same _List_ of objects all with the same names:
+
+```
+$ awk '/name:/{print}' file/quota-*.yml
+file/quota-large.yml:    name: quota
+file/quota-large.yml:    name: burst-quota
+file/quota-large.yml:    name: limits
+file/quota-medium.yml:    name: quota
+file/quota-medium.yml:    name: burst-quota
+file/quota-medium.yml:    name: limits
+file/quota-small.yml:    name: quota
+file/quota-small.yml:    name: burst-quota
+file/quota-small.yml:    name: limits
+```
+
+The _profiles_ are differentiated using labels and annotations:
+
+```
+$ awk /'labels:/{getline; print}' file/quota-*.yml
+      quota-tier: Large
+      quota-tier: Large
+      quota-tier: Large
+      quota-tier: Medium
+      quota-tier: Medium
+      quota-tier: Medium
+      quota-tier: Small
+      quota-tier: Small
+      quota-tier: Small
+
+$ awk /'annotations:/{getline; print}' file/quota-*.yml
+      openshift.io/quota-tier: Large
+      openshift.io/quota-tier: Large
+      openshift.io/quota-tier: Large
+      openshift.io/quota-tier: Medium
+      openshift.io/quota-tier: Medium
+      openshift.io/quota-tier: Medium
+      openshift.io/quota-tier: Small
+      openshift.io/quota-tier: Small
+      openshift.io/quota-tier: Small
+```
```

Comment:
Seems you had a different directory layout when you wrote this and we need to `s/file/files/` in the above code blocks.